config/routes.rb

```diff
@@ -9,7 +9,14 @@
   end
 
   namespace :admin do
-    resources :basic_projects, path: 'projects'
+    resources :basic_projects do
```

Comment:
This will conflict on merge because I changed the path thing. Can you please add `path: 'basic-projects'` and that should fix it.